NetSDK/Services/Client/Classes/ConfigurationOptions.cs

```diff
@@ -8,5 +8,27 @@ namespace Splitio.Services.Client.Classes
     public class ConfigurationOptions
     {
         public string LocalhostFilePath { get; set; }
+        public int? FeaturesRefreshRate { get; set; } 
+        public int? SegmentsRefreshRate { get; set; } 
+        public bool RandomizeRefreshRates { get; set; } 
+        public int? ImpressionsRefreshRate { get; set; }
+        public int? MaxImpressionsLogSize { get; set; }  
+        public long? ConnectionTimeout { get; set; } 
+        public long? ReadTimeout { get; set; } 
```

Comment:
Since we have a nice C# object now, what do you think about to use _TimeSpan_ in place of integers? it will avoid confusions like _"is it in seconds or milliseconds?"_